cli/server.go

```diff
@@ -80,6 +82,8 @@ func server() *cobra.Command {
 		oauth2GithubClientSecret         string
 		oauth2GithubAllowedOrganizations []string
 		oauth2GithubAllowSignups         bool
+		telemetryEnabled                 bool
```

Comment:
Keeping with what you told me the other day this should just be `telemetry`, correct?

Reply:
Good catch